utils/helper.py

```diff
@@ -24,3 +28,18 @@ def calculate_earning(task):
     rate = task.rate_per_hour
     total_earning = time * rate
     return total_earning
+
+
+def my_scheduled_task():
+    current_time = datetime.date.today() + datetime.timedelta(days=1)
+    tasks = TaskModel.objects.filter(due_date=current_time)
+    lst = [
+        send_emails
+            (
+            subject="Last Date of Submission",
+            message=f"Tomorrow is the last date of submitting your task {task.task}",
+            recipient=task.assigned_to
+        )
+        for task in tasks if task.status != TaskModel.COMPLETE
```

Comment:
Do we really need this loop here?

Reply:
and make sure to update the requirements every time you add a new package or something.